server/api/events.js

```diff
@@ -109,3 +113,66 @@ router.post('/', async (req, res, next) => {
     next(err)
   }
 })
+
+/*
+      * Event Init - Creates interactions for each player, joins pairs and adds a round number
+      */
+router.get('/:eventId/interactions', async (req, res, next) => {
```

Comment:
This is creating interactions so it should be a post.

There should also be a GET for this that GETS things.